mmdet/models/backbones/efficientnet.py

```diff
@@ -0,0 +1,282 @@
+import mmcv
+import torch
+import torch.nn as nn
+import torch.utils.checkpoint as cp
+from mmcv.cnn import build_conv_layer, build_norm_layer
+
+from ..builder import BACKBONES
+from ..utils.activations import MemoryEfficientSwish
+from ..utils.inverted_residual import InvertedResidual
+
+
+def drop_path(x, drop_prob=0., training=False):
```

Comment:
MMCV already provide `drop_path`. We should use that in MMCV if they look the same.
see https://github.com/open-mmlab/mmcv/blob/c47c9196d067a0900b7b8987a8e82768edab2fff/mmcv/cnn/bricks/drop.py#L9